dev/test/typescript.ts

```diff
@@ -33,6 +33,7 @@ import Precondition = FirebaseFirestore.Precondition;
 import SetOptions = FirebaseFirestore.SetOptions;
 import Timestamp = FirebaseFirestore.Timestamp;
 import Settings = FirebaseFirestore.Settings;
+import {FirestoreDataConverter} from '@google-cloud/firestore';
```

Comment:
Does `import FirestoreDataConverter from FirebaseFirestore.FirestoreDataConverter` not work?

Also, if it does - please sort the imports.

Reply:
Forgot to export it in the `index.ts` file, but now it does.